docs/sources/metainfo/component.xml

```diff
@@ -789,6 +789,48 @@
 				</listitem>
 			</varlistentry>
 
+			<varlistentry id="tag-requires-recommends-control">
+				<term>&lt;control/&gt;</term>
+				<listitem>
+				<para>
+					This item type can be used to recommend or require certain ways a user can control the software. This usually maps to certain methods
```

Comment:
Which is it, _recommend_ or _require_?

Reply:
maybe just change the wording to `and`?

Reply:
It's either recommend or require, depending on which of the two tags the item is declared in. I think and is actually equally valid to "or" here, I'll have a look at the rest of the specification and change all of these occurrences in one go to what makes the most sense (never really thought about it that much).